src/util/formattingNodeToHtml.ts

```diff
@@ -3,17 +3,38 @@ import _ from 'lodash'
 import stripStyleAttribute from './stripStyleAttribute'
 
 /** Strip span and encode a <i> or <b> element as HTML. */
+const replaceFontToSpan = (node: Element) => {
+  if (node.tagName === 'font') {
+    node.tagName = 'span'
+    node.attributes = node.attributes.map(attr => {
+      return attr.key === 'style'
+        ? { key: attr.key, value: attr.value }
+        : {
+            key: 'style',
+            value: `${attr.key}: ${attr.value}`,
+          }
+    })
+  }
+  return node
+}
+
+/** Converts a formatting node to HTML. */
 const formattingNodeToHtml = (node: Element) => {
   const content: string = node.children.reduce((acc, child) => {
     return acc + (child.type === 'text' ? child.content : child.type === 'comment' ? '' : formattingNodeToHtml(child))
   }, '')
-
+  replaceFontToSpan(node)
   if (node.tagName === 'span') {
-    const styleAttribute = _.find(node.attributes, { key: 'style' })
-    if (!styleAttribute) {
+    const styleAttributes = _.filter(node.attributes, { key: 'style' })
+    if (!styleAttributes) {
       return content
     }
-    const strippedStyleAttribute = stripStyleAttribute(styleAttribute.value)
+    let strippedStyleAttribute = ''
+    styleAttributes.forEach(styleAttribute => {
+      const strippedAttribute = stripStyleAttribute(styleAttribute.value)
+      if (strippedAttribute.length > 0) strippedStyleAttribute += strippedAttribute
+    })
```

Comment:
Please use a map and join instead of mutation. Mutation is generally not recommended.